_DENA/resource/helpers.py

```diff
@@ -103,6 +106,73 @@ def query_tracks(engine: 'Engine', start_date: str, end_date: str,
     return data
 
 
+def query_adsb(adsb_files: List[str],  start_date: str, end_date: str,
```

Comment:
query_adsb and query_early adsb seem identical. Can we make the code a little DRYer by maybe just adding an if else statements like (please excuse my pseudocode):

```
if start_date year is 2019 or before:
    adsb = EarlyAdsb(adsb_files)
else:
     adsb = Adsb(adsb_files)
```